addons/website_payment/views/snippets/s_donation.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_donation" name="Donation">
+    <section class="s_donation pt32 pb32 o_cc o_cc1" style="position: relative;" data-oe-shape-data='{"shape":"web_editor/Origins/02","flip":[]}'>
+        <div class="o_we_shape o_web_editor_Origins_02"/>
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="s_donation_column col-lg-7 pt16 pb16">
+                    <h2>Make a Donation</h2>
+                    <p>Help us to work with <b>local communities</b> to protect our <b>planet</b>.</p>
+                    <div class="s_donation_prefilled_buttons">
+                        <button id="s_donation_btn_1" class="s_donation_btn btn btn-outline-primary btn-lg mb-2 mr-1 o_not_editable" contenteditable="false" data-donation-value="10">10</button>
+                        <button id="s_donation_btn_2" class="s_donation_btn btn btn-outline-primary btn-lg mb-2 mr-1 o_not_editable" contenteditable="false" data-donation-value="25">25</button>
+                        <button id="s_donation_btn_3" class="s_donation_btn btn btn-outline-primary btn-lg mb-2 mr-1 o_not_editable" contenteditable="false" data-donation-value="50">50</button>
+                        <button id="s_donation_btn_4" class="s_donation_btn btn btn-outline-primary btn-lg mb-2 mr-1 o_not_editable" contenteditable="false" data-donation-value="100">100</button>
+                    </div>
+                    <div class="s_donation_input_container my-4" data-donation-layout="freeAmount">
+                        <div class="row s_col_no_resize s_col_no_bgcolor">
+                            <label for="s_donation_amount_input" class="col-sm-4 col-form-label">Enter your own amount</label>
+                            <div class="col-sm-4">
+                                <input id="s_donation_amount_input" type="number" min="5" class="form-control" aria-label="Amount"/>
+                                <small class="form-text text-muted">
+                                    <i>Minimum Donation: <span class="s_donation_minimum_value font-weight-bold" contenteditable="false">5</span></i>
+                                </small>
+                            </div>
+                        </div>
+                    </div>
```

Comment:
Could there be a way to avoid duplicating the `000.xml` chunks ?
Maybe have an initial phase on snippet drop that renders according to the content according to the default settings ? (Maybe by using the `onBuilt()` method in `options.js`)

Reply:
> Could there be a way to avoid duplicating the `000.xml` chunks ?
> Maybe have an initial phase on snippet drop that renders according to the content according to the default settings ? (Maybe by using the `onBuilt()` method in `options.js`)

seeing what has been done for the cookie bar I think there is no solution to avoid duplicating it